charts/argo-rollouts/templates/dashboard/clusterrole.yaml

```diff
@@ -17,8 +17,10 @@ rules:
       - get
       - list
       - watch
+      {{- if not .Values.dashboard.readonly }}
       - update
       - patch
+      {{- end }}
```

Comment:
Looks like that is the only place we have the `create` verb and I don't think we actually may need it

Reply:
I agree, `create` verb in dashboard's clusterrole is not necessary. As per testing, create experiment and analysisrun only triggers when there is a new version of rollouts that includes it in the steps. Otherwise, it doesn't really create experiment and analysisrun when you restart the rollouts.

should we completely remove `create` verb in dashboard's clusterrole?